resources/monitoring/profile-production.yaml

```diff
@@ -17,13 +17,29 @@ prometheus:
               storage: 20Gi
 
 prometheusOperator:
+  namespaces:
+    releaseNamespace: "true"
   resources:
     requests:
       cpu: "10m"
       memory: "128Mi"
     limits:
       cpu: "500m"
 
+prometheus-istio:
+  server:
+    resources:
+      requests:
+        cpu: 200m
+        memory: 768Mi
+      limits:
+        cpu: 1000m
+        memory: 8Gi
+
+defaultRules:
+  disabled:
+    PrometheusNotConnectedToAlertmanagers: "true"
```

Comment:
That whole block should not get in as it is specific to the SAP setup and will break compatibility from OS perspective